airflow-bi/pyproject.toml

```diff
@@ -13,6 +13,7 @@ requests = "^2.25.1"
 boto3 = ">=1.0.0"
 s3fs = "^0.5.2"
 oauth2client = "^4.1.3"
+bs4 = "^0.0.1"
```

Comment:
Can you use `beautifulsoup4` instead? Although the build is reproducible while we keep the lock file, it's not that much if we don't. From [bs4](https://pypi.org/project/bs4/) package.

> This is a dummy package managed by the developer of Beautiful Soup to prevent name squatting. The official name of PyPI’s Beautiful Soup Python package is beautifulsoup4. This package ensures that if you type pip install bs4 by mistake you will end up with Beautiful Soup.

So that 0.0.1 is basically acting as a `latest`.

Reply:
Replaced the packages! Rebuilding the image...